backend/accounts.go

```diff
@@ -58,6 +58,10 @@ const accountsHardLimit = 5
 // AccountsList is an accounts.Interface slice which implements a lookup method.
 type AccountsList []accounts.Interface
 
+// KeystoresAccountsListMap is a map where keys are keystores' fingerprints and values are
```

Comment:
the keys are Keystore object pointers, not fingerprints. I think it would be better to make the key the actual fingerprint (as bytes if possible, otherwise as hex string), as it's the only thing needed, and then one does not have to worry about pointers (multiple different pointers could point to the same keystore object, for example).

Reply:
Addressed in #2609